pages/en/lb4/Creating-components.md

```diff
@@ -90,12 +90,13 @@ A note on binding names: In order to avoid name conflicts, always add a unique p
 Provider's `value()` method can be asynchronous too:
 
 ```js
+import {Provider} from '@loopback/context';
 const request = require('request-promise-native');
 const weatherUrl =
   'http://samples.openweathermap.org/data/2.5/weather?appid=b1b15e88fa797225412429c1c50c122a1'
 
-export class CurrentTemperatureProvider {
-  async value() {
+export class CurrentTemperatureProvider implement Provider<number>{
```

Comment:
typo here in the code snippet, should be "implements"

Reply:
horrible typo :(